README.md

```diff
@@ -1992,5 +1993,10 @@ The API will return a JSON payload with the `Content-Type` response header set t
 No such header is required to query the API.
 
 
+### Installing as binary
+You can download Gatus as a binary using the following command:
+
+`go install github.com/TwiN/gatus/v5@latest`
```

Comment:
Could you use \`\`\` instead?
e.g.
```
\```
go install ...
\```
```
(Without the backslashes)


I'm at kubecon this week & I don't have my computer; couldn't figure out how to add a suggestion and commit it myself to your fork, sorry. There are other occurrences in the README if you need a reference.

Reply:
Done @TwiN